src/validation.cpp

```diff
@@ -348,7 +348,9 @@ void CChainState::MaybeUpdateMempoolForReorg(
     // previously-confirmed transactions back to the mempool.
     // UpdateTransactionsFromBlock finds descendants of any transactions in
     // the disconnectpool that were added back and cleans up the mempool state.
-    m_mempool->UpdateTransactionsFromBlock(vHashUpdate);
+    const uint64_t ancestor_count_limit = gArgs.GetIntArg("-limitancestorcount", DEFAULT_ANCESTOR_LIMIT);
```

Comment:
any reason we can't access these fields via `m_mempool->m_limit_ancestors` and `m_mempool->m_limit_ancestor_size`?

Reply:
i think the main reason is that there is no such field on m_mempool?

Reply:
(it's a field on MemPoolAccept not on the mempool iteself)

Reply:
ok that would do it 🤦